playground/terraform/infrastructure/variables.tf

```diff
@@ -167,5 +165,8 @@ variable "gke_name" {
 
 variable "gke_location" {
   description = "Location of GKE cluster"
-  default     = "us-central1-a"
+}
+
+variable "pg_location" {
```

Comment:
Personally I found the `pg` in this `pg_location` name confusing initially.  May we consider variable names in the language of the cloud environment?  For example, `gke_location` is clear in that it refers to the location of the GKE node.  In some infrastructure-as-code modules, I've even seen just the use of `region` as it is conventionally understood that region refers to the Compute Engine region.  Then, when applying to the terraform GKE module, the location property is simply assigned to the `var.region` variable.

Reply:
Fixed